com.gigitux.youp.json

```diff
@@ -0,0 +1,40 @@
+{
+  "app-id": "com.gigitux.youp",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.36",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "youp",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=session-bus",
```

Comment:
Do you really need the whole bus? Using `--talk-name` is usually preferred

Reply:
Mmm, I'm actually not sure if this is necessary.


I need webkit to be able to send notifications. Is it necessary?